### PR TITLE
feat: turn on cloud mode

### PR DESCRIPTION
**Pull request by drdelambre**
Closes #

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -52,12 +52,15 @@
     "cy:dev-oss": "source ../monitor-ci/.env && CYPRESS_dexUrl=OSS CYPRESS_baseUrl=https://$INGRESS_HOST:$PORT_HTTPS cypress open --config testFiles='{oss,shared}/**/*.*'",
     "generate": "export SHA=27c725d02cbabaf21be8bcecd43b2871840829bc && export REMOTE=https://raw.githubusercontent.com/influxdata/openapi/${SHA}/ && yarn generate-meta",
     "generate-local": "export REMOTE=../openapi/ && yarn generate-meta",
-    "generate-meta": "yarn currentApi && yarn notebooks && yarn unity && yarn flows && yarn managedFunctions && yarn annotations",
+    "generate-meta": "if [ -z \"${CLOUD_URL}\" ]; then yarn generate-meta-oss; else yarn generate-meta-cloud; fi",
+    "generate-meta-oss": "yarn oss-api && yarn notebooks && yarn unity && yarn flows && yarn managedFunctions && yarn annotations",
```
Comment (Asalem1) [inline package.json:56]: are annotations enabled for OSS?

**Review by Asalem1 — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (drdelambre) [reply package.json:56]: according to the existing infra settings, yes!

Comment (glinton) [reply package.json:56]: oss doesn't have any annotations logic at all last i knew

**Comment by drdelambre** [comment]
seems like we need to add a (unit/lint)-(cloud/oss) distinction as well

**Comment by ui-ci-bot-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 50 | :red_circle:
accessibility | 96 | :green_circle:
seo | 90 | :green_circle:
pwa | null | :hankey:
best-practices | 100 | :green_circle:


[Compare your scores with master branch](https://influxdb.aws.influxdata.io/orgs/844910ece80be8bc/dashboards/073c4b95f1089000?lower=now%28%29%20-%202d&vars%5Blighthouse_score_branch%5D=1654)

**Pull request merged**